src/shapes/group.class.js

```diff
@@ -105,7 +105,11 @@
         //  we need to preserve object's center point in relation to canvas and apply group's transform to it
         var inv = invertTransform(this.calcTransformMatrix());
         this.forEachObject(function (object) {
-          var center = transformPoint(object.getCenterPoint(), inv);
```

Comment:
Why did you remove this???
It is correct

Reply:
i put back the original code because i have a failing UT.
I want all the test to pass and the figure out what is going on.
Now i have 2 completely unrelated UTs failing only in browser

Reply:
> master: ![image](https://user-images.githubusercontent.com/34343793/160274911-97ee19a1-9ab0-4e9c-b475-a990682ebd29.png)
> 
> before your fix: ![image](https://user-images.githubusercontent.com/34343793/160274998-f9d2e888-9ee9-49a2-ae94-3f8e5aa7f975.png)
> 
> with your fix: ![image](https://user-images.githubusercontent.com/34343793/160274966-1cf10dd7-174e-4943-a83d-66de63998302.png)
> 
> I'll push a visual test for this

Reply:
@asturur the UT is wrong

Reply:
Something is still wrong
![image](https://user-images.githubusercontent.com/34343793/160278987-cf5bd538-f31e-47f4-bcd0-a3ffb5ffeea7.png)

Reply:
I have found the bug.
Give me a few minutes

Reply:
fixed 847284f7b93580ebe3b9fbe165bf209417ae009e